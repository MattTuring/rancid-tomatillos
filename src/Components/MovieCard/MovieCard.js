import React from 'react'
import { Link } from 'react-router-dom';

const MovieCard = ({title, poster, rating, id}) => {
  return (
  <article className='movie-card'>
    <h2>{title}</h2>
    <div className='movie-img' style={{backgroundImage: `url(${poster})`}}/>
    <div className='rating'>RATING: {rating}</div>
    <Link to={"/movies/" + id}>
      <button>MORE</button>
    </Link>
  </article>
  )
}

export default MovieCard
