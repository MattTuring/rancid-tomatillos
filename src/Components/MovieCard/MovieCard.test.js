import React from 'react';
import { shallow } from 'enzyme';
import MovieCard from './MovieCard';

describe('MovieCard', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<MovieCard
      title="Foster Movie"
    />);

    expect(wrapper).toMatchSnapshot();
  });
    
});