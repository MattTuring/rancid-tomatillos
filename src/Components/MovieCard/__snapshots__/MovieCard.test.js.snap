// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard should match the snapshot with all data passed in correctly 1`] = `
<article
  className="movie-card"
>
  <h2>
    FOSTER MOVIE
  </h2>
  <div
    className="movie-img"
    style={
      Object {
        "backgroundImage": "url(undefined)",
      }
    }
  />
  <div
    className="rating"
  >
    AVERAGE RATING: 
  </div>
  <Link
    to="/movies/undefined"
  >
    <button>
      MORE
    </button>
  </Link>
</article>
`;
